spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/LocalArmeriaPortTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.spring.LocalArmeriaPortTest.TestConfiguration;
+
+/**
+ * Tests for {@link LocalArmeriaPort}.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = TestConfiguration.class)
+@ActiveProfiles({ "local", "autoConfTest" })
+@DirtiesContext
+public class LocalArmeriaPortTest {
+
+    @SpringBootApplication
+    @Import(ArmeriaOkServiceConfiguration.class)
+    static class TestConfiguration {
+    }
+
+    @Inject
+    private Server server;
+
+    @LocalArmeriaPort
```

Comment:
Not sure if it's simple or not, but how about adding a test case where this annotation is used on method?

Reply:
Fixed 568291b.

And I also wrote additional tests for https. :wink: